snooty/main.py

```diff
@@ -202,6 +203,7 @@ def on_update(
             "ast": page.ast.serialize(),
             "source": page.source,
             "static_assets": checksums,
+            "created_at": datetime.utcnow(),
```

Comment:
The `on_update_metadata()` function performs a similar operation on the `metadata` collection (where we store things like TOCs, breadcrumbs, and other inter-site data). Although the collection is much smaller, I imagine we ought to perform a similar culling operation on it so that we expire _all_ documents associated with a build.

So basically we can add this line there, too!

Reply:
Logical! I admit I had no idea that collection existed :D Done!